extensions/amp-next-page/validator-amp-next-page.protoascii

```diff
@@ -19,14 +19,15 @@ tags: {  # amp-next-page
   extension_spec: {
     name: "amp-next-page"
     version: "0.1"
+    version: "1.0"
     version: "latest"
   }
   attr_lists: "common-extension-attrs"
 }
-tags: {  # amp-next-page: json config
+tags: {  # amp-next-page: JSON Config
   html_format: AMP
   tag_name: "SCRIPT"
-  spec_name: "amp-next-page extension .json configuration"
+  spec_name: "AMP-NEXT-PAGE > SCRIPT"
```

Comment:
Why the change? Just SCRIPT is ambiguous as to what type of script it is. If you like changing to this, then  `AMP-NEXT-PAGE > SCRIPT[type=application/json]`.